lib/oxidized/model/stoneos.rb

```diff
@@ -0,0 +1,38 @@
+class Stoneos < Oxidized::Model
+#Hillstone Networks StoneOS software
+
+  prompt /^\r?[\w.()-]+[#>](\s)?$/
+  comment '# '
+  
+  expect /^\s.*--More--.*$/ do |data, re|
+   send ' '
+   data.sub re, ''
+  end
+ 
+  cmd :all do |cfg|
+    cfg = cfg.lines.to_a[1..-2].join
+  end
+
+  cmd :secret do |cfg|
+    cfg
+  end
+
+  cmd 'show configuration running' do |cfg|
```

Comment:
Since the input is not modified in any way, this block can instead be expressed as a one line `cmd 'show configuration running'`